src/foundry/client/apps/sidebar/tabs/combat-tracker.d.mts

```diff
@@ -36,7 +36,7 @@ declare global {
     /**
      * Return an array of Combat encounters which occur within the current Scene.
      */
-    get combats(): Document.Stored<Combat.ConfiguredInstance>[];
+    get combats(): Document.Stored<Combat.Implementation>[];
```

Comment:
Could you also do a pass for `Document.Stored<(ActiveEffect|...)(\.Implementation)?>` -> `$1.Stored`?